pkg/internal/corecli/corecli.go

```diff
@@ -59,3 +57,16 @@ func extractPlugin(fs afero.Fs) (string, error) {
 
 	return pluginFile.Name(), nil
 }
+
+// InstallPlugin installs the core plugin bundled with the CLI.
```

Comment:
The scope of the function doesn't seem limited to the core plugin, is it?

Reply:
You mean the name `InstallPlugin`? The package name `corecli` should already indicate that functions in it are related to the core plugin.